sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -176,6 +184,15 @@ def delete_directory(self, **kwargs):
         :keyword int timeout:
             The timeout parameter is expressed in seconds.
         :return: None
+
+        .. admonition:: Example:
```

Comment:
I'm sure you did, but please make sure you generate docs locally and they seem fine.

Reply:
Thanks for reminding me that! 
 Just noticed some of them look weird file:///C:/Users/xiafu/Downloads/azure-storage-file-datalake%20(1)/azure.storage.filedatalake.html